### ISSUE TITLE
missing include <stdexcept>

### ISSUE THREAD

**Issue by Trass3r** [simonask/rx-ranges#37]
https://godbolt.org/z/3HB9h_

**Comment by Kijewski** [comment]
Instead of adding the include I'd vote for "demoting" the runtime checks to assertions.

```diff
-        if (n == 0) {
-            throw std::runtime_error("in_groups_of_exactly(0) is not allowed");
-        }
+        RX_ASSERT(n > 0);
```

**Comment by Trass3r** [comment]
True, it's actually the only usage, preventing compilation with `-fno-exceptions`.

**Comment by simonask** [comment]
Yeah, it should have been an assertion.

**Comment by simonask** [comment]
Fixed by #44
